extensions/guacamole-auth-jdbc/modules/guacamole-auth-jdbc-postgresql/src/main/java/org/apache/guacamole/auth/postgresql/PostgreSQLEnvironment.java

```diff
@@ -47,6 +47,21 @@
      */
     private static final int DEFAULT_PORT = 5432;
 
+    /**
+     * The default defaultStatementTimeout (in seconds),
+     * if POSTGRESQL_DEFAULT_STATEMENT_TIMEOUT is not specified.
+     * Default to null (no timeout)
+     * https://mybatis.org/mybatis-3/configuration.html
+     */
+    private static final String DEFAULT_DEFAULT_STATEMENT_TIMEOUT = "null";
+
+    /**
+     * The default socketTimeout (in seconds), if POSTGRESQL_SOCKET_TIMEOUT is not specified.
+     * Default to 0 (no timeout)
+     * https://jdbc.postgresql.org/documentation/head/connect.html
+     */
+    private static final int DEFAULT_SOCKET_TIMEOUT = 0;
+
```

Comment:
Is there some reason why you've implemented one of these as a `String` and the other as an `int`?  I think I would stick with `int` to be consistent with how other timeouts are implemented.  Also, one uses `null` for no timeout, the other uses 0 - can we align on that, or does the underlying code note allow for that?

Reply:
Yes, I was trying to keep same as what the backend takes. But I see your point - consistency at this level would be good rather than exposing the underlying differences.